src/elements/OcMenuItem.vue

```diff
@@ -29,6 +27,12 @@ export default {
       default: "",
     },
   },
+  computed: {
+    $_ocMenuItem_menuItemClass() {
```

Comment:
no need to apply a constant via a computed property - right?

Reply:
Right. Fixed